src/screens/Accounts/index.js

```diff
@@ -51,6 +52,33 @@ class Accounts extends NavigatorComponent<Props & Actions & AccountsState> {
     });
   }
 
+  static showResetPasscodeSuccess = async (navigator: Navigator) => {
+    alert('successResetPassword', [
+      {
+        name: 'confirm',
+        onPress: async () => {
+          if (Platform.OS === 'ios') {
+            await navigator.dismissModal();
+            navigator.pop();
```

Comment:
`navigator.pop()` line is the same for both platforms, so can be moved outside of the `if` statement.

Reply:
done